boards/arm/stm32h7/stm32h747i-disco/src/stm32_dma_alloc.c

```diff
@@ -0,0 +1,117 @@
+/****************************************************************************
+ * boards/arm/stm32h7/stm32h747i-disco/src/stm32_dma_alloc.c
+ *
+ *   Copyright (C) 2016 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
```

Comment:
New files should have Apache license. Please copy the text from [here](https://github.com/apache/incubator-nuttx/blob/master/sched/sched/sched_getcpu.c)

Reply:
No 3rd party licensed code is permitted in Apache repositorys.  This cannot be merged without the Apache license header with no author and no copyright.

Reply:
When you contribute software to NuttX you are literally giving it to the Apached Software Foundation and must have the Apache license.

Reply:
license is now changed to Apache